frontend/src/components/shared/Modal/Modal.tsx

```diff
@@ -136,6 +137,10 @@ function Modal(props: ModalProps): ReactElement {
     Dialog: {
       style: {
         border: `1px solid ${colors.fadedText10}`,
+        borderTopLeftRadius: "0.25rem",
```

Comment:
I think it is better to use the `radii.md` here from the theme. I think you can get `radii` in the line above from `useTheme`:

https://github.com/streamlit/streamlit/blob/0de5d8e37369be9b186683c70a766ccafcd247fb/frontend/src/components/shared/Modal/Modal.tsx#L129

Reply:
Good point 👍 I've changed it to use `radii` here 👍